src/lib.rs

```diff
@@ -332,6 +334,8 @@ impl NavigatorBuilder {
 
         let led = Led::new();
 
+        let leak = Pin::new(27);
```

Comment:
//Define leakage sensor Pin

Reply:
Why do you mean by it ? I'm following the same pattern as in the LED initialization. It's already clear that 27 is the pin for the leak detection in the code.

Reply:
just to have some overall explanation, which sensor.

Reply:
> just to have some overall explanation, which sensor. The LEDs are self-explained, but leakage sensors need to attach an extra gadget to GPIO.

That's documented on the read_leak function.

Reply:
ok.